### PR TITLE
#538 - reload page after logout

### PR DESCRIPTION
**Pull request by samrichca**
Reload the page after logout because the page wasn't updating after logout on develop.

### CODE REVIEW

**Comment by ciaranschutte** [comment]
why are there two logout functions? 
There seems to be some parts of duplicated logic between auth hook and _app which maybe going back in the direction we came from. Can we constrain everything to the auth hook? Some minor tweaking is needed of course but probably better to do while we're early in this. eg. pass a path to the auth hook logout to account for a custom path

**Comment by samrichca** [comment]
> why are there two logout functions? There seems to be some parts of duplicated logic between auth hook and _app which maybe going back in the direction we came from. Can we constrain everything to the auth hook? Some minor tweaking is needed of course but probably better to do while we're early in this. eg. pass a path to the auth hook logout to account for a custom path

(Discussed on Slack) I have to share auth logic between app.tsx and the auth hook because I wasn't able to put all of the auth logic into the auth hook, and get a working solution, in the timeframe required. This is good feedback though and something to keep in mind.

I've revised my PR to make a utility for logging out which is shared by app.tsx & the auth hook.

File: pages/_app.tsx [hunk 1]
```diff
@@ -106,7 +98,7 @@ const App = ({
         logout();
       }
     };
-    handleAuth();
+    if (!ctx.query?.session_expired) handleAuth();
```
Comment (anncatton) [inline pages/_app.tsx:101]: just a code style quibble...can `handleAuth()` be put inside a `{}` for the if statement? i always find this one line style harder to read

**Review by anncatton — APPROVED** [review]

File: pages/_app.tsx (hunk 1 above)
Comment (samrichca) [reply pages/_app.tsx]: Done!

**Pull request merged**